### DISCUSSION TITLE
TVM News+

### DISCUSSION THREAD

**Discussion by darrencycheung** [iptv-org/community#819] [Show and tell]
### Channel ID

TVMnewsPlus.mt

### Stream URL

https://dist4.tvmi.mt/eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOjI0NDczLCJhdWQiOiIxMjUuMjMwLjIyNS4zMyIsImV4cCI6MTY2NDcxOTQ5OH0.u4dmnyCGFAvrumkD4D2mbulBnf1JquOLfF6cUt0-OP0/live/2/master.m3u8

### Notes

Maybe not 24/7

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Stream does not work: please close.

**Comment by freearhey** [comment]
https://dist4.tvmi.mt/eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOjI0NDczLCJhdWQiOiIxMjUuMjMwLjIyNS4zMyIsImV4cCI6MTY2NDcxOTQ5OH0.u4dmnyCGFAvrumkD4D2mbulBnf1JquOLfF6cUt0-OP0/live/2/master.m3u8 (offline)
<img width="836" alt="image" src="https://user-images.githubusercontent.com/7253922/232236737-e97f2767-b2e2-4920-acca-d26d4606ac3f.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/mt_smash_plus.m3u#L48-L49
